build-system/tasks/coverage-map.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+const argv = require('minimist')(process.argv.slice(2));
+const fs = require('fs');
+const log = require('fancy-log');
+const puppeteer = require('puppeteer');
+const {exec} = require('../common/exec');
+
+const coverageJsonName = argv.json || 'out.json';
+const testUrl =
+  argv.url || 'http://localhost:8000/examples/everything.amp.html';
+
+async function compile() {
+  await exec(`gulp dist`);
+  // The question: How to cd to 'amphtml/dist' from anywhere in 'amphtml'?
```

Comment:
The last issue is running this task from anywhere in the `amphtml` repository. I'd like to change the working directory to `amphtml/dist`, but I don't quite know how to do it.

Reply:
`exec` ends up calling `spawnSync`, which accepts an `options` object allowing a `cwd` option:
https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options

Reply:
@rsimha told me that `gulp` automatically redirects to the root repository, which means this actually isn't an issue in the first place.